modules/process/SV/SomaticRunManta.nf

```diff
@@ -47,5 +45,23 @@ process SomaticRunManta {
     ${outputPrefix}.manta.vcf.gz
   mv Manta/results/variants/somaticSV.vcf.gz.tbi \
     ${outputPrefix}.manta.vcf.gz.tbi
+
+
+  # Filter variants that have low supporting reads in the tumor or high supporting reads in the normal
+  # PR = discordant reads
+  # SR = split reads
+  bcftools view \\
+    -s ${idTumor},${idNormal} \\
+    ${outputPrefix}.manta.vcf.gz | \\
+  bcftools filter \\
+    --soft-filter tumor_read_supp -m + \\
+    -e "FORMAT/PR[0:1] < 5 | FORMAT/SR[0:1] < 2" | \\
+  bcftools filter \\
+    --soft-filter normal_read_supp -m + \\
+    -e "FORMAT/PR[1:1] > 0 | FORMAT/SR[1:1] > 0" | \\
+  bcftools view --output-type z > \\
+    ${outputPrefix}.manta.filtered.vcf.gz
```

Comment:
Just want to make sure:

Are `${outputPrefix}.manta.vcf.gz` and `${outputPrefix}.manta.filtered.vcf.gz` having the same number of data lines? Difference are in the filtering annotations?

Same question for delly.

Reply:
yes, the `--soft-filter` allows you to keep the variant, but updates the FILTER value.